.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,9 +42,9 @@
 - [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
 - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
 - [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
-- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
+- [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
```

Comment:
You don't have to do this, but it would be nice if you could indent this item in so that it appears as a sub-bullet of the item above and then say "Or, I have added manual tests ... ". This item and the one above it are mutually exclusive so it would be nice if that could be represented in some way.

Reply:
Done